setup.cfg

```diff
@@ -55,6 +55,7 @@ classifiers =
 include_package_data = True
 install_requires =
   importlib-metadata >= 4; python_version < '3.8'
```

Comment:
Dropping support for older Pythons means that this will never be hit. With that, `python_requires` would need to be updated. And the Trove classifiers too.

However, I'd like to ask you to move the changes related to changing the Python support matrix to a separate PR.